enf-reducer/gather-junctors.metta

```diff
@@ -1,48 +1,29 @@
-(: gatherJunctors (-> Tree Tree Tree))
-(= (gatherJunctors $currentNode $centerNode)
-      (let* (
-              ($currentNodeType (getNodeType $currentNode))
-              ($centerNodeType (getNodeType $centerNode))
-              ($currentLeft (getChild $currentNode L))
-              ($currentRight (getChild $currentNode R))
-              ((TreeNode (Value $valueName $constraint $currentNodeType) $guardSet $children) $currentNode)
+; a function to merge two tuples (A B C) (D E F) ==> (A B C D E F)
+(= (concat-tuple $xs $ys)

```

Comment:
It will also be cool if you can add test cases for this function using the `assertEqual` in a utility file or even in the gatherJucntor file?

Reply:
And I think this will be a very useful helper function, it will be better if moved somewhere else. Also don't be shy to use some symbols as a function name like `++` or something. We already have a function named as `++` right now but you can find some other symbol that will be easy to understand and even defined as an infix operator instead of prefix like so
```scheme
(: ++ (-> Atom Atom Atom))
(= ($xs ++ $ys) ...)
```
This will make it will better readable whenever someone sees the code.